SQL/0000-00-01-Permission.sql

```diff
@@ -113,3 +113,27 @@ LOCK TABLES `user_perm_rel` WRITE, `permissions` READ;
 INSERT INTO `user_perm_rel` (userID, permID) SELECT DISTINCT 1, permID FROM permissions;
 /*!40000 ALTER TABLE `user_perm_rel` ENABLE KEYS */;
 UNLOCK TABLES;
+
+CREATE TABLE `permission_categories` (
```

Comment:
This is me just being nitpicky, can we change `permission_categories` to `permission_roles`. We already have a table called `permissions_catergory` which can get confusing